client/packages/system/src/Patient/VaccinationCards/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { FC, useEffect } from 'react';
+import {
+  TableProvider,
+  createTableStore,
+  useBreadcrumbs,
+  createQueryParamsStore,
+  useParams,
+  InlineSpinner,
+  useTranslation,
+  DataTable,
+  useColumns,
+  NothingHere,
+  LocaleKey,
+} from '@openmsupply-client/common';
+import {
+  usePatientVaccineCard,
+  VaxCardData,
+} from '../../api/hooks/usePatientVaccineCard';
+
+export const VaccinationCardComponent: FC = () => {
+  const t = useTranslation('dispensary');
+  const { patientId = '', programEnrolmentId = '' } = useParams();
+  const { setCustomBreadcrumbs } = useBreadcrumbs();
+
+  const {
+    query: { data, isLoading },
+  } = usePatientVaccineCard(patientId, programEnrolmentId);
+
+  const tableData = buildTableData(data);
+
+  const columns = useColumns<DoseRowData>([
+    {
+      key: 'age',
+      label: 'label.age',
+      sortable: false,
+      accessor: ({ rowData }) => `${rowData.age} months`, // TO-DO: express in years/months
+    },
+    {
+      key: 'label',
+      label: 'label.dose',
+      accessor: ({ rowData }) => rowData?.label,
+    },
+    {
+      key: 'status',
+      label: 'label.status',
+      accessor: ({ rowData }) => (rowData?.status ? t(rowData?.status) : ''),
+      // Cell: DotCell,
+    },
+    {
+      key: 'suggestedDate',
+      label: 'label.suggested-date',
+      accessor: ({ rowData }) => rowData?.suggestedDate,
+      // Cell: DateCell, TO-DO
+    },
+    {
+      key: 'dateGiven',
+      label: 'label.date-given',
+      accessor: ({ rowData }) => rowData?.dateGiven,
+      // Cell: DateCell, TO-DO
+    },
+  ]);
+
+  useEffect(() => {
+    if (data)
+      setCustomBreadcrumbs({
+        1: data?.vaccineCardItems?.patient?.name ?? '',
+        2: t('label.vaccination-card'),
+        3: data?.vaccineCardItems?.programName,
```

Comment:
Clicking on the vaccination card part of the breadcrumb:
![Screenshot 2024-09-11 at 11 50 58 AM](https://github.com/user-attachments/assets/4d0040e4-95e6-41d0-be93-6403c95e3807)

Takes you here:
![Screenshot 2024-09-11 at 11 50 53 AM](https://github.com/user-attachments/assets/c6aa8aed-2118-4f1b-b2e8-08ef24fa729c)

I think either need to 
- disable the link on that crumb (had a quick look, looks like we have a `disabled` field in the breadcrumb state, but haven't implemented a nice way to set that yet, could make a refactor issue? maybe can do something like what was done in [ContactTrace](https://github.com/msupply-foundation/open-msupply/blob/4663-patient-vax-card/client/packages/system/src/ContactTrace/DetailView/DetailView.tsx#L115-L149))
- or add a redirect back to patient detail view if they end up on the `{patientId}/vaccine-card` route